tests/examples/hello/src/sum.cc

```diff
@@ -0,0 +1,34 @@
+#include "hello_world.h"
+#include "legate_library.h"
+
+namespace hello {
+
+class SumTask : public Task<SumTask, SUM> {
+public:
+  static void cpu_variant(legate::TaskContext &context) {
+    auto &input = context.inputs()[0];
+    auto input_shape = input.shape<1>(); // should be a 1-Dim array
+    auto in = input.read_accessor<float, 1>();
+
+    auto n = input_shape.volume();
+    float total = 0;
+    for (size_t i = 0; i < n; ++i) {
+      total += in[i];
+    }
+
+    using Reduce = Legion::SumReduction<float>;
+    auto sum = context.reductions()[0].reduce_accessor<Reduce, true, 1>();
```

Comment:
Adding type & size checks here will make it clear this is a singleton `Store` we're reducing into:
```
Store& output = context.reductions()[0];
assert(output.code() == FLOAT_LT);
assert(output.dim() == 1);
assert(output.shape() == Rect<1>(0,0));
```